packages/ebook-tools/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE='https://sourceforge.net/projects/ebook-tools/'
```

Comment:
Since this is an indirect dependency it would be good to mention what package requires it.
```suggestion
# Dependency for kf6-kfilemetadata
TERMUX_PKG_HOMEPAGE='https://sourceforge.net/projects/ebook-tools/'
```